routing/missioncontrol.go

```diff
@@ -93,6 +93,10 @@ type NodeResults map[route.Vertex]TimedPairResult
 // since the last failure is used to estimate a success probability that is fed
 // into the path finding process for subsequent payment attempts.
```

Comment:
nit: the commit message mentions "persistance" which in my mind means writing to disk. So i'd use "propagate" instead